graphein/protein/features/sequence/sequence.py

```diff
@@ -38,7 +38,7 @@ def molecular_weight(
 """
 
 
-def molecular_weight(input, seq_type="protein"):
+def molecular_weight(input, data: Optional = None, seq_type="protein"):
```

Comment:
Should this line be the following?

```suggestion
def molecular_weight(input, data: Optional = None, seq_type: str = "protein"):
```

Also, `seq_type` might need to be restricted to whatever is acceptable to `SeqUtils.molecular_weight`, I think. You could do a separate `validate_seq_type` function in `sequence.py` to help with validating that the input is done right.